### PR TITLE
Addon updates

### PR DESCRIPTION
**Pull request by heitbaum**
- btrfs-progs: update to 6.5 and addon (2)
- docker: update to 24.0.6 and addon (2)
  - cli: update to 24.0.6
  - containerd: update to 1.7.5
  - Moby: update to 24.0.6
  - runc: update to 1.1.9
- inadyn: update to 2.11.0 and addon (1)
- prometheus-node-exporter: update to 1.6.1 and addon (1)
- syncthing: update to 1.24.0 and addon (1)
- system-tools: addon (3)
  - bottom: update to 0.9.6
  - inotify-tools: update to 4.23.8.0
  - libgpiod: update to 2.0.2
  - mc: pcre2 and XDG change
  - nmon: update to 16p and HSTS
- apache-ant: update to 1.10.14
- go: update to 1.21.1
- icu: update to 73.2
- libvpx: update to 1.13.0
- libzip: use libzip.org url and dont build for zstd
- protobuf: update to 21.12

### CODE REVIEW

File: packages/addons/addon-depends/protobuf/package.mk [hunk 1]
```diff
@@ -2,11 +2,11 @@
 # Copyright (C) 2016-present Team LibreELEC (https://libreelec.tv)
 
 PKG_NAME="protobuf"
-PKG_VERSION="3.20.3"
-PKG_SHA256="e51cc8fc496f893e2a48beb417730ab6cbcb251142ad8b2cd1951faa5c76fe3d"
+PKG_VERSION="21.12"
```
Comment (CvH) [inline packages/addons/addon-depends/protobuf/package.mk:5]: is there a reason we switch the version ?
likewise they play version bingo at protobuf but should we stay at the current versioning ?
they have 3 different versions with the same hash for some reason
isn`t 24.2 the latest version and everything else is "something else" ? 

image

File: packages/addons/addon-depends/system-tools-depends/nmon/package.mk [hunk 2]
```diff
@@ -2,11 +2,11 @@
 # Copyright (C) 2016-present Team LibreELEC (https://libreelec.tv)
 
 PKG_NAME="nmon"
-PKG_VERSION="16n"
+PKG_VERSION="16p"
```
Comment (CvH) [inline packages/addons/addon-depends/system-tools-depends/nmon/package.mk:5]: version change with same hash ?

**Review by CvH — CHANGES_REQUESTED** [review]

File: packages/addons/addon-depends/protobuf/package.mk (hunk 1 above)
Comment (heitbaum) [reply packages/addons/addon-depends/protobuf/package.mk:5]: 21 is actually 3 based on my navigation. Which is why I went with this. Also jumping to 4 or 22 - changes the packaging - and as you say is not the same product.

File: packages/addons/addon-depends/system-tools-depends/nmon/package.mk (hunk 2 above)
Comment (heitbaum) [reply packages/addons/addon-depends/system-tools-depends/nmon/package.mk:5]: Fixed sha and pkgurl

**Review by CvH — APPROVED** [review]

**Pull request merged**